background.js

```diff
@@ -147,7 +151,7 @@ async function moveToBugzilla(data) {
   const bugzillaId = response.id;
 
   if (!bugzillaId) {
-    throw new Error("Could not create bugzilla bug.");
+    throw new Error(`Could not create bugzilla bug: ${JSON.stringify(response)}`);
```

Comment:
Why do we need to log this? What is in that object exactly, can you show an example via pastebin? Ditto for the `result` object above

Reply:
(Just to make sure we don't log anything private by accident)

Reply:
This might include the error response from bugzilla - eg, this is how I leaned my token needed SSO privs.